### DISCUSSION TITLE
基于 Git 仓构建微服务应用 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#972] [Announcements]
# 基于 Git 仓构建微服务应用 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/wizard/create-git-based-ms/

**Comment by zgfh** [comment]
我们这里的demo是否可以以一个开源项目地址（如github或gitee）为demo，能够按着文档一步步操作就能搞定demo，这样对新手会更友好

**Reply by jzhupup** [reply]
好，后续我们优化下
